ETL/appsettings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "ConnectionStrings": {
+        "Host": "192.168.1.1",
+        "UserName": "animeMaster17",
```

Comment:
Need to make these secrets in teh future

Reply:
those are fake values